docs/guides/user-guide/importing-projects.md

```diff
@@ -43,14 +43,14 @@ Oftentimes, the first thing you do when opening a project is to launch a command
 
 Usually, these kinds of commands exist in the `scripts` section of your project's `package.json` file and you would manually type `npm run dev` to execute them.
 
-Using StackBlitz, you can provide an [npm script](https://docs.npmjs.com/cli/v8/using-npm/scripts) to run automatically when the editor opens with the `terminal` query parameter:
+Using StackBlitz, you can provide an [npm script](https://docs.npmjs.com/cli/v8/using-npm/scripts) to run automatically when the editor opens with the `startScript` query parameter:
 
-`stackblitz.com/fork/github/{gh_username}/{repo_name}?terminal={npm_script_name}`
+`stackblitz.com/fork/github/{gh_username}/{repo_name}?startScript={npm_script_name}`
 
 :::tip Example
 The following URL will open the `vitesse` repository of the `antfu` user, install the npm dependencies, and run `npm run dev` command in the terminal:
 
-[stackblitz.com/github/antfu/vitesse?terminal=dev](http://www.stackblitz.com/github/antfu/vitesse?terminal=dev)
+[stackblitz.com/github/antfu/vitesse?startScript=dev](https://stackblitz.com/github/antfu/vitesse?startScript=dev)
```

Comment:
Linking to `http://www.stackblitz.com` or `http://stackblitz.com` instead of `https://stackblitz.com` triggers a redirection, and this redirection loses query parameters from the original URL.

Reply:
should we fix that?